frontend/src/app/pages/Leaderboards/index.js

```diff
@@ -6,5 +6,5 @@ import { Subtitle, Box } from './../../../styles/blocks'
 export const LeaderboardsPage = () =>
   <Box Margin="10px" Padding="10px">
     <Subtitle textAlign="center">Hall of Fame</Subtitle>
-    <Leaderboards maxItems={Number.MAX_SAFE_INTEGER} />
+    <Leaderboards maxItems={Number.MAX_SAFE_INTEGER} showFilters={true} />
```

Comment:
is it a LEADERBOARD or LEADERBOARD-S-? pls check with a native

Reply:
Leaderboards are plural and as the page displays leaderboard for several criteria I decided for the `-S` version. However, I will ask some native on his opinion.

Reply:
Upon consulting will be refactored to LEADERBOARD...

Reply:
Fixed [here](https://github.com/salsita/foosball-rating/pull/92/commits/b14aa874e76bee5974082fd1bd250e0673533d99)